index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+</head>
+
+<body>
+    <header>
+        <a href="#">
+            <h3>My Logo</h3>
+        </a>
+        <nav>
+            <img src="./images/list.svg" alt="Logo" />
+        </nav>
+    </header>
+    <section class="introduction_section">
+        <div class="intro_title">
+            <h1>I'm Obedi<br> Glad to see you!</h1>
+        </div>
+        <div class="intro_descript">
+            <p> I’m a software developer! I can help
+                you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you
+                need coded, don’t hestiate to contact me.
+            </p>
+        </div>
+        <div class="connect_links">
+            <a href="#">LET'S CONNECT</a>
```

Comment:
- [ ] [OPTIONAL] Kindly use a `hx` tag for this as it does not link anywhere within or outside the page